src/main/java/com/fcgl/madrid/points/service/UserTrophyService.java

```diff
@@ -1,22 +1,61 @@
 package com.fcgl.madrid.points.service;
 
 import com.fcgl.madrid.points.model.UserTrophy;
+import com.fcgl.madrid.points.model.response.GetUserTrophyResponse;
 import com.fcgl.madrid.points.repository.UserTrophyRepository;
+import com.fcgl.madrid.points.model.InternalStatus;
+import com.fcgl.madrid.points.model.StatusCode;
+import com.fcgl.madrid.points.model.ExceptionUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.TransactionSystemException;
 
+import io.github.resilience4j.circuitbreaker.CallNotPermittedException;
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import java.lang.Exception;
+import java.lang.Throwable;
+import java.lang.StringBuilder;
+
+import java.util.Set;
 import java.util.List;
+import java.util.ArrayList;
 
 @Service
 public class UserTrophyService {
-    private UserTrophyRepository userTrophyRepository;
+	private UserTrophyRepository userTrophyRepository;
+
+	@Autowired
+	public UserTrophyService(UserTrophyRepository userTrophyRepository) {
+		this.userTrophyRepository = userTrophyRepository;
+	}
+
+	public List<UserTrophy> findAll() {
+		return userTrophyRepository.findAll();
+	}
+
+	@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")
+	public GetUserTrophyResponse getUserTrophiesById(Long userId) {
+		return new GetUserTrophyResponse(InternalStatus.OK,	userTrophyRepository.getUserTrophiesById(userId));
+	}
+
 
-    @Autowired
-    public UserTrophyService(UserTrophyRepository userTrophyRepository) {
-        this.userTrophyRepository = userTrophyRepository;
-    }
+	/**
+	 *
+	 * @param ex Exception
+	 * @return ResponseEntity<InternalStatus>
+	 */
+	@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")
```

Comment:
You can delete this annotation. The only annotation should be in `getUserTrophiesById()`